CONTRIBUTING.md

```diff
@@ -0,0 +1,2 @@
+### Contributing to CodeJoust
+You are free to contribute to the `main` branch of CodeJoust. If you notice any issues on our website, feel free to add an issue or submit your own pull request. If you have any questions, contact us at [support@codejoust.co](support@codejoust.courl).
```

Comment:
```suggestion
You are free to contribute to the `main` branch of CodeJoust. If you notice any issues on our website, feel free to add an issue or submit your own pull request. If you have any questions, contact us at [support@codejoust.co](mailto:support@codejoust.co).
```

Reply:
After our discussion, I'll remove the license so there is simply [no license](https://choosealicense.com/no-permission/) so our work is under exclusive copyright.